_inc/lib/debugger/class-jetpack-cxn-test-base.php

```diff
@@ -277,15 +291,18 @@ public static function skipped_test( $name = 'Unnamed', $message = false ) {
 	 * @since 7.1.0
 	 * @since 7.3.0 Added $action for resolution action link, $severity for issue severity.
 	 *
-	 * @param string $name Test name.
-	 * @param string $message Message detailing the failure.
-	 * @param string $resolution Optional. Steps to resolve.
-	 * @param string $action Optional. URL to direct users to self-resolve.
-	 * @param string $severity Optional. "critical" or "recommended" for failure stats. "good" for passing.
+	 * @param string      $name Test name.
+	 * @param string      $message Message detailing the failure.
+	 * @param string      $resolution Optional. Steps to resolve.
+	 * @param string      $action Optional. URL to direct users to self-resolve.
+	 * @param string      $severity Optional. "critical" or "recommended" for failure stats. "good" for passing.
+	 * @param string      $label Optional. The label to use instead of the test name.
+	 * @param string      $action_label Optional. The label for the action url instead of default 'Resolve'.
+	 * @param string|bool $description Optional. An HTML description to override resolution.
 	 *
 	 * @return array Test results.
 	 */
-	public static function failing_test( $name, $message, $resolution = false, $action = false, $severity = 'critical' ) {
+	public static function failing_test( $name, $message, $resolution = false, $action = false, $severity = 'critical', $label = false, $action_label = 'Resolve', $description = false ) {
```

Comment:
Maybe we can pull `Resolve` out of there so it can still be translated?

```suggestion
	public static function failing_test( $name, $message, $resolution = false, $action = false, $severity = 'critical', $label = false, $action_label = '', $description = false ) {
		if ( empty( $action_label ) ) {
			$action_label = __( 'Resolve', 'jetpack' );
		}
```